oti4me/more-recipes
# client side review recipe

#### What does this PR do?

client side review recipe

#### Description of Task to be completed?

- add review compoment
- add review list modal
- add add new review modal
- add get review action
- add action to add user review
- refactor getHeader.js file to a function

#### How should this be manually tested?

- Clone online repository
- Navigate to the cloned directory
- Run npm install
- visit localhost:3000
- Signup/signin
- click on a recipe from the recent recipe list
- click on the review button to see list of reviews
- click on the add review button
- add a review using the popup modal

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

- story type: feature
- story id: 153951261
- story title: client side review recipe

**Screenshots (if appropriate)**
